src/storages/memory_storage/mod.rs

```diff
@@ -38,8 +38,7 @@ impl Store<Key> for MemoryStorage {
     async fn fetch_schema(&self, table_name: &str) -> Result<Option<Schema>> {
         self.items
             .get(table_name)
-            .map(|item| item.schema.clone())
-            .map(Ok)
+            .map(|item| Ok(item.schema.clone()))
```

Comment:
Q: Is there any reason to change this?

Reply:
Nothing special reason, `cargo-tarpaulin` marks two `map` as not reached codes, so I just merged two independent lines into an one.